yt/fields/field_info_container.py

```diff
@@ -667,4 +667,19 @@ def check_derived_fields(self, fields_to_check=None):
             dfl = filtered_dfl
 
         self.ds.derived_field_list = dfl
+        self._set_linear_fields()
         return deps, unavailable
+
+    def _set_linear_fields(self):
+        """
+        Sets which fields use linear as their default scaling in Profiles and
+        PhasePlots. Default for all fields is set to log, so this sets which
+        are linear.  For now, set linear to geometric fields: position and
+        velocity coordinates.
+        """
+        log_prefixes = ['', 'velocity_', 'particle_position_', 'particle_velocity_']
+        coords = ['x', 'y', 'z']
+        log_fields = [prefix+coord for prefix in log_prefixes for coord in coords]
+        for field in self.ds.derived_field_list:
+            if field[1] in log_fields:
+                self[field].take_log = False
```

Comment:
The variable got me confused. Shouldn't it be instead

```suggestion
        non_log_prefixes = ('', 'velocity_', 'particle_position_', 'particle_velocity_')
        coords = ('x', 'y', 'z')
        non_log_fields = [prefix+coord for prefix in log_prefixes for coord in coords]
        for field in self.ds.derived_field_list:
            if field[1] in non_log_fields:
                self[field].take_log = False
```